README.md

```diff
@@ -52,10 +58,17 @@ There are currently three configuration options:
 ## Public Functions
 Trackomatic.js makes certain internal utility functions public by way of the global trackomatic object. The following functions are available under trackomatic.util:
 
+- capitalize
+- createAnchor
 - createCookie
-- readCookie
-- getURLParam
+- createNavigationHandler
+- debounce
+- getHost
+- getLink
 - getPathname
-- proper
+- getViewportSize
+- keyCode
```

Comment:
`keyCode` seems like an outlier here, since there's no verb in the name, but what it does is so simple that it's not a big deal. Hmmm.